### PR TITLE
Dashboard/srushti

### PR DESCRIPTION
**Pull request by srushtisalke**

### CODE REVIEW

**Comment by srushtisalke** [comment]
Please find video link here:
https://www.loom.com/share/3441b0e29bef4ff79f8778710f01ba74?sid=ed2f4475-c72c-464d-9d78-163d4ae97872

**Comment by srushtisalke** [comment]
@aina21 could you please review it ?

**Comment by aina21** [comment]
Please make sure to resolve the existing conflicts.

File: src/styles/teamDetails.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.Team-Member-container {
+  /* background-color: aqua; */
```
Comment (aina21) [inline src/styles/teamDetails.css]: remove it

File: src/styles/team.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.container-section {
+  background-color:  #ffb98a;
+}
+.container-heading .logo{
+  /* padding-left: 4em; */
```
Comment (aina21) [inline src/styles/team.css]: remove it

File: src/pages/Team.jsx [hunk 3]
```diff
@@ -1,57 +1,40 @@
...
+import DeleteTeam from '../component/DeleteMember'
+import TeamMembers from '../component/TeamDetails'
+import MembersList from '../component/MembersList'
+import '../styles/team.css'
+
+function Team(){
+return(
+  <>
+  <section className="welcome-team">
+    {/* <h1>Welcom Back, {team.team_name}</h1> */}
```
Comment (aina21) [inline src/pages/Team.jsx]: remove it

File: src/pages/Team.jsx [hunk 4]
```diff
@@ -1,57 +1,40 @@
...
+    <MembersList></MembersList>
```
Comment (aina21) [inline src/pages/Team.jsx]: `<MembersList/>`

File: src/pages/Team.jsx [hunk 5]
```diff
@@ -1,57 +1,40 @@
...
+  </section>
+  <div className="container-section">
+    <section className="left-section">
+      <section className="TeamMember">
+        <TeamMembers></TeamMembers>
```
Comment (aina21) [inline src/pages/Team.jsx]: same here, if you don't need to include anything between the opening and closing tags of a component, you can use a self-closing tag instead.

**Comment by srushtisalke** [comment]
@aina21  I have done changes and resolved conflicts as well. 
Please find all changes and request in commit section.

File: src/pages/Team.jsx (hunk 3 above)
Comment (srushtisalke) [reply src/pages/Team.jsx]: done

File: src/pages/Team.jsx (hunk 4 above)
Comment (srushtisalke) [reply src/pages/Team.jsx]: done

File: src/pages/Team.jsx (hunk 5 above)
Comment (srushtisalke) [reply src/pages/Team.jsx]: done

File: src/styles/teamDetails.css (hunk 1 above)
Comment (srushtisalke) [reply src/styles/teamDetails.css]: done

File: src/component/MembersList.jsx [hunk 6]
```diff
@@ -1,10 +1,14 @@
...
 import { apiPath } from '../api';
+import '../styles/memberList.css';
 
 const MembersList = ({teamId}) => {
   const [members, setMembers] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [team, setTeam] = useState(null);
+  const { code } = useParams();
 
   useEffect(() => {
```
Comment (aina21) [inline src/component/MembersList.jsx:13]: as we fetch data for both team and member in the useEffect, try to use promise.all if possible to improve effeciency. something like

```js
const [teamData, membersData] = await Promise.all([
          fetchTeam(),
          fetchMembers()
        ]);
```

File: src/component/MembersList.jsx [hunk 7]
```diff
@@ -27,7 +48,15 @@ const MembersList = ({teamId}) => {
   return (
     <div className="containerList">
       <div className="titleContainer">
-        <h4>Here is a list of your team members!</h4>
+        <section>
```
Comment (aina21) [inline src/component/MembersList.jsx]: Shouldn’t the condition also include the `<section>` element itself? If the team does not exist, the `<section>` element will still be added to the DOM without any value, which might not be ideal.

**Review by aina21 — APPROVED** [review]

**Pull request merged**